### PR TITLE
Rework of the rsyslog formula

### PR DESCRIPTION
**Pull request by elemoine**
This PR fixes a few things in the formula, and adds support for Heka.

### CODE REVIEW

**Review by simonpasquier — APPROVED** [review]

**Comment by fpytloun** [comment]
Yes, I like this approach - much better than the one with linux formula.
Whole functionality needs to be tested anyway to verify status of this formula.

**Comment by elemoine** [comment]
> Yes, I like this approach - much better than the one with linux formula.
> Whole functionality needs to be tested anyway to verify status of this formula.

Thanks for the review. I'll let you know when this is ready for final review.

**Comment by elemoine** [comment]
This is now ready for review. I'd recommend reviewing this commit by commit, instead of looking at the full diff.

File: rsyslog/meta/heka.yml [hunk 1]
```diff
@@ -25,7 +25,14 @@ log_collector:
       decoder: "syslog_decoder"
       splitter: "TokenSplitter"
     {%- endfor %}
-  {%- endif %}
+    {%- endif %}
+    syslog_haproxy:
```
Comment (simonpasquier) [inline rsyslog/meta/heka.yml:29]: Naive question but shouldn't it be in the HAProxy formula?

**Comment by simonpasquier** [comment]
Otherwise great work!

**Pull request merged**